x-pack/plugins/actions/server/monitoring/register_cluster_collector.ts

```diff
@@ -40,8 +43,7 @@ export function registerClusterCollector({
     },
     fetch: async () => {
       const [_, pluginStart] = await core.getStartServices();
-      const nowInMs = +new Date();
-      const { docs: overdueTasks } = await pluginStart.taskManager.fetch({
+      const results = await pluginStart.taskManager.aggregate({
```

Comment:
I tried injecting an error here, because I'm afraid of ES runtime field issues.  The `api/monitoring_collection` request returns a 500, with a stack trace in the console.  Not sure if that's appropriate or not, and if it isn't, is there some way to "report" an error to monitoring like that?  

Could be something we do in another issue ...

Reply:
I agree the route should not return a 500. Added a try/catch to the `fetch` function to return empty stats in the case of failure here: https://github.com/elastic/kibana/pull/135824/commits/10315fe8485b635fcb20ceec2043757ebeaab113

I can create an issue for how we want to report these errors in a more holistic fashion

Reply:
Added very lightweight functional tests in [this commit](https://github.com/elastic/kibana/pull/135824/commits/4dd712d45df65072ad4f3b19d80d6f898c7d0a1d) to ensure the route doesn't return a 500